---
int64[] object_class
float32[] position_x
float32[] position_y
float32[] position_z
